### PR TITLE
Added support for the original Contour Design ShuttlePRO

### PR DESCRIPTION
**Pull request by 1div0**
I am glad to see Contour Design ShuttlePRO finally working after 7even years since https://github.com/1div0/Shuttle

Please review and consider merge into master proposed changes. Thank you so much.

### CODE REVIEW

File: gtk2_ardour/ardour.sh.in [hunk 1]
```diff
@@ -61,8 +61,9 @@ export VAMP_PATH=@LIBDIR@/vamp${VAMP_PATH:+:$VAMP_PATH}
 
 if [ $# -gt 0 ] ; then
     case $1 in
-        -g|--gdb) GDB=gdb; shift ;;
+        -d|--ddd) DEBUGGER=ddd; shift ;;
```
Comment (x42) [inline gtk2_ardour/ardour.sh.in:64]: `-d` is an option for Ardour and should not be used by the wrapper script.

```
  -d, --disable-plugins       Disable all plugins (safe mode)
```

Comment (1div0) [reply gtk2_ardour/ardour.sh.in:64]: Thank you. I will fix that. Was not aware.

**Comment by pauldavisthefirst** [comment]
gtk2_ardour/opts.cc

**Comment by x42** [comment]
I've rebased and merged the main part of this (ShuttlePRO). 6.0-77 630a3d07d2

As for the debugger, that should be a separate commit.

gtk2_ardour/ardbg also supports `lldb` (for the clang-crowd) already.

I think we should only support other debuggers when running Ardour from the source-tree.
When asking end-users to run the installed binary in a debugger, all documentation refers to gdb, and that is also the debugger that we ship with windows binaries.

**Comment by 1div0** [comment]
Danke.

**Comment by x42** [comment]
Final part was added as 6.2-245 6e67eba5bb

**Comment by 1div0** [comment]
I've found one mistake. Original Contour Design ShuttlePRO has two buttons less than second generation. However, currently I have no time to fix that. Maybe l8r next month.

**Comment by johannes-mueller** [comment]
Basically a shuttleXpress behaves like a ShuttlePRO v2 only that some buttons are never pressed (because they are not present in the shuttleXpress). The only place where this difference is taken into account is in the UI. We prevent the users to configure actions to buttons that there device doesn't have.

An alternative would be that we let users configure all the three devices separately, so one for each device.

In my practical work I am using both, the ShuttlePRO v2 and the shuttleXpress, but I have one configuration that works for both.

As I don't have an old ShuttlePRO, I don't know which are the buttons that the old ShuttlePRO doesn't have. Can you check this?

**Comment by 1div0** [comment]
This is the original ShuttlePRO
IMG_20200913_153614

**Comment by johannes-mueller** [comment]
Then it's probably buttons number 14 and 15 that are missing.

**Pull request closed without merging**